logger/utils_test.go

```
@@ -0,0 +1,183 @@
+}
+
+func TestGetLoggerRejectsInaccessibleLogFile(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("file modes do not reliably prevent writes on Windows")
+	}
+
+	logPath := filepath.Join(t.TempDir(), "azurehound.log")
+	if err := os.WriteFile(logPath, nil, 0400); err != nil {
+		t.Fatal(err)
+	}
+	defer os.Chmod(logPath, 0600)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Chmod` is not checked

(errcheck)